src/translations/en.json

```
@@ -599,7 +594,7 @@
     "NetworkModificationTree": "Network modification tree",
     "RestoreModifications": "Restore modifications",
     "RestoreModificationText": "Please select modifications to restore",
-    "DeleteModificationText": "Are you sure you want to permanently delete {numberToDelete} modifications ?",
+    "DeleteModificationText": "{numberToDelete, plural, =0 {Zero modification to delete.} =1 {One modification will be permanently deleted.} other {# modifications will be permanently deleted.}}",
```

maybe "No modification [...]" instead of "Zero modification [...]"?
```suggestion
    "DeleteModificationText": "{numberToDelete, plural, =0 {No modification to delete.} =1 {One modification will be permanently deleted.} other {# modifications will be permanently deleted.}}",
```